docs/defects-api.yml

```diff
@@ -198,6 +260,92 @@ components:
       type: array
       items:
         $ref: "#/components/schemas/imNumber"
+
+    ivaDefects:
+      type: array
+      items:
+        $ref: "#/components/schemas/ivaDefect"
+
+    ivaDefect:
+      type: object
+      properties:
+        sectionNumber:
+          type: string
+        sectionDescription:
+          type: string
+        vehicleTypes:
```

Comment:
I can definitely understand why euVehicleCategories would be an array, ie m2 & m3. But would the vehicleType ever be an array?